gradle/libs.versions.toml

```diff
@@ -14,7 +14,7 @@ junit = "5.8.2"
 
 # buildSrc
 indra = "3.1.3"
-lombok = "1.18.30"
+lombok-version = "1.18.30"
```

Comment:
doesn't really matter but i would just leave it as `lombok` since nothing else has that suffix and `lombok-plugin` is unique for the plugin

Reply:
The issue is if I leave it as `lombok` it won't work because it collides with lombok-plugin, which makes it impossible to do `libs.versions.lombok.get()`, it's a gradle quirk.

Reply:
oh okay